arch/arm/core/aarch64/reset.c

```diff
@@ -36,6 +38,28 @@ void z_arm64_el_highest_init(void)
 	isb();
 }
 
+enum el3_next_el {
+	EL3_TO_EL2,
+	EL3_TO_EL1_NO_EL2,
+	EL3_TO_EL1_SKIP_EL2
+};
+
+static inline enum el3_next_el arm64_el3_get_next_el(void)
```

Comment:
@jharris-intel since you have to push again anyway to unstuck CI, can you rename this to `el3_get_next_el()`?